README.md

```diff
@@ -98,7 +98,7 @@ vars:
     shopify_timezone: "America/New_York" # Replace with your timezone
 ```
 
-> **Note**: This will only **numerically** convert timestamps to your target timezone. They will however have a "UTC" appended to them. This is a current limitation of the dbt-date `convert_timezone` [macro](https://github.com/calogica/dbt-date#convert_timezone-column-target_tznone-source_tznone) we leverage.
+> **Note**: This will only **numerically** convert timestamps to your target timezone. They will however have a "UTC" appended to them. This is a current limitation of the dbt-date `convert_timezone` [macro](https://github.com/calogica/dbt-date#convert_timezone-column-target_tznone-source_tznone) we have leveraged and replicated [here](https://github.com/fivetran/dbt_shopify_source/tree/main/macros/fivetran_date_macros/fivetran_convert_timezone.sql) with minimal modifications.
```

Comment:
Now that this is internal to the package, I wonder if we should create a feature request to update the macro to remove this irregular behavior (not something worth spending time on in this PR though).

Reply:
Good idea, I'll make a FR

Reply:
https://github.com/fivetran/dbt_shopify_source/issues/99